### ISSUE TITLE
NMR1d workflow

### ISSUE THREAD

**Issue by namratakale** [phnmnl/phenomenal-h2020#31]
the tutorial needs updating

**Comment by korseby** [comment]
where is it located?

**Comment by DSchober** [comment]
I think Reza and DanielJ are the best persons to ask here.

**Comment by DSchober** [comment]
I don't think that much has changed since Birmingham. Not sure what in particular really deserves a full update ?

**Comment by djcomlab** [comment]
According to the wiki page history, @rsalek and @pcm32 authored the tutorial.

**Comment by korseby** [comment]
The workflow itself hasn't changed. As far as I know, we only need to fix the `Coming soon` in `STEP 2: Runing the NMR Galaxy workflow` and feed it with some nice screenshots of the new statistics Daniel Jacobs provided.

**Comment by rsalek** [comment]
A nice delegation of a job ending up with the two  people that left ;) Can help, if needed

**Comment by djcomlab** [comment]
Well, I don't know why I was delegated this at all to begin with as I'm nothing to do with this workflow or this tutorial or anything NMR-related...

**Comment by rsalek** [comment]
> Well, I don't know why I was delegated this at all to begin with as I'm nothing to do with this workflow or this tutorial or anything NMR-related...
I think your ID was mistaken for Daniel Jacob! ...sharing the djco... part ;)

**Comment by DSchober** [comment]
exactly. Just sloppy assigning based on faulty pattern matching.
